miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplay.kt

```diff
@@ -68,17 +89,31 @@ internal class RealMiniAppDisplay(
         }
     } else false
 
+    @Suppress("LongMethod")
     private suspend fun provideMiniAppWebView(context: Context): MiniAppWebView =
         miniAppWebView ?: withContext(Dispatchers.Main) {
-            miniAppWebView = MiniAppWebView(
-                context = context,
-                basePath = basePath,
-                miniAppInfo = miniAppInfo,
-                miniAppMessageBridge = miniAppMessageBridge,
-                miniAppNavigator = miniAppNavigator,
-                hostAppUserAgentInfo = hostAppUserAgentInfo,
-                miniAppCustomPermissionCache = miniAppCustomPermissionCache
-            )
+            if (miniAppUrl != null) {
+                miniAppWebView = MiniAppWebView(
```

Comment:
This can be replace with `MiniAppHttpWebView`. See [here](https://github.com/minh-rakuten/android-miniapp/blob/03e78f86f5130608ce4f94e210b47ae6f4beec72/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplay.kt).